src/connection/mc_auth_logic.erl

```diff
@@ -52,27 +52,35 @@ scram_sha_1_auth(Socket, Database, Login, Password, SetOpts) ->
     scram_first_step(Socket, Database, Login, Password, SetOpts)
```

Comment:
Minor nit: the typespec says `scam_sha_1_auth` returns a `boolean()`, but now it may return `true | ok | {error, _}`

Reply:
Thanks!